store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -645,14 +657,81 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
 
   private def updateRollups(pgu: PGSecondaryUniverse[SoQLType, SoQLValue],
                             originalCopyInfo: Option[TruthCopyInfo],
-                            copyInfo: TruthCopyInfo) = {
+                            copyInfo: TruthCopyInfo,
+                            rollupDirty: RollupDirty) = {
+    if (rollupDirty.affectAll || originalCopyInfo.isEmpty) {
+      updateRollupsSlow(pgu, originalCopyInfo, copyInfo, rollupDirty)
+    } else {
+      updateRollupsFast(pgu, originalCopyInfo, copyInfo, rollupDirty),
+    }
+  }
+
+  private def updateRollupsSlow(pgu: PGSecondaryUniverse[SoQLType, SoQLValue],
+                                originalCopyInfo: Option[TruthCopyInfo],
+                                copyInfo: TruthCopyInfo,
+                                rollupDirty: RollupDirty): Unit = {
```

Comment:
`rollupDirty` is unused.

Reply:
I am aware but am trying to keep the signature identical.  Well...I don't insist.